doc/boardState.mer

```diff
@@ -0,0 +1,9 @@
+---
+title: "Board states"
```

Comment:
You could get away without the title, as it is redundant with the README heading. Same feedback on the title in `classDiagram.mer`.